qt-bugs@ issue : none
bugs.kde.org number : none
applied: no
author: Dirk Mueller <mueller@kde.org>

fix array underrun. should be in 3.3.8 according to TT


--- src/codecs/qrtlcodec.cpp	(revision 628194)
+++ src/codecs/qrtlcodec.cpp	(working copy)
@@ -106,7 +106,7 @@ static bool to8bit(const QChar ch, QCStr
 	    *rstr += (char)ch.cell();
 	    converted = TRUE;
 	} else if( ch.cell() < 0xA0 ) {
-	    *rstr += (char)unicode_to_heb_00[ch.cell() - 0xA0];
+	    *rstr += (char)unicode_to_heb_00[ch.cell() - 0x80];
 	    converted = TRUE;
 	}
     }
